packages/shared/styles/helpers/_colors.scss

```diff
@@ -0,0 +1,21 @@
+@each $color, $value in $colors {
+  // Atomic utility classes for text and bg colors
+  .text-#{$color} {
+    color: $value !important;
+  }
+  .bg-#{$color} {
+    background-color: $value !important;
+  }
+
+  // General color modifiers
+  // May be overwritten by specific component modifiers
+  .color-#{$color} {
```

Comment:
I think we might need to approach it differently. For example, if the chosen background color is `accent` we have a `text-on-accent` variable for this, same for dark, light, etc (so we need to add classes manually here). In the proposed implementation everything will be in different shades of the same color if I understand it correctly, right?

Reply:
```scss
$c-text-on-primary: color-yiq($c-primary) !default;
```
Agree, something like the above?

I just think we should use `color-yiq` by default to make customization easier, in this case the user can just set `$c-primary` without the need of set `$c-text-on-primary`.

Reply:
Yes, I think setting this default is a good solution that will work for most of the users. Others can override

Reply:
ok done, I've just renamed the variable, I think `$c-text-on-primary` was too big, so I'm using `$c-on-primary`